test/prog/dftb+/scc/2H2O_nonvariational/dftb_in.hsd

```diff
@@ -0,0 +1,44 @@
+Geometry = GenFormat {
+6 C
+ O H
+   1   2     -.499735     -.006429      .000000
+   2   1    -1.467047     -.147444      .000000
+   3   1     1.392547      .133948      .000000
+   4   2    -1.876914      .729873      .000000
+   5   2     1.737135     -.325150     -.779513
+   6   2     1.737135     -.325150      .779513
+}
+
+Driver = {}
+
+Hamiltonian = DFTB {
+  SCC = Yes
+  VariationalEnergy = No
```

Comment:
This seems not to match the feature as implemented above (althoug I like this approach more as it would made the logic much easier...) and the test fails, if executed. Due to a typo in `tests` (and a bug in `autotest2`, which apparently causes non-existent tests to succeed) it was not reported as failure.